core/templates/pages/profile-page/profile-page.component.ts

```diff
@@ -101,80 +101,90 @@ export class ProfilePageComponent {
     this.DEFAULT_PROFILE_PICTURE_URL = this.urlInterpolationService
       .getStaticImageUrl('/general/no_profile_picture.png');
     this.loaderService.showLoadingScreen('Loading');
-    this.profilePageBackendApiService.fetchProfileDataAsync()
```

Comment:
NOTE: The git diff in this file makes it look very complicated. But I just unindented a bunch of lines and moved the profile image part into its own `fetch` call. So the output is 2 promises.